examples/simple/simple.tf

```diff
@@ -10,10 +10,12 @@ module "acs" {
 
 module "rds" {
   //  source = "github.com/byu-oit/terraform-aws-rds?ref=v0.2.0"
```

Comment:
change the version to v0.2.1 or something like that (and change the version in the other example)

Reply:
Will do.